src/main/resources/data/minecolonies/tags/items/blacksmith_ingredient.json

```diff
@@ -1,7 +1,8 @@
-{
-  "replace": false,
-  "values": [
-    "#forge:nuggets",
-    "#forge:ingots"
-  ]
-}
\ No newline at end of file
+{
+  "replace": false,
+  "values": [
+    "#forge:nuggets",
+    "#forge:ingots",
+    "#forge:storage_blocks"
+  ]

```

Comment:
Hmm, this allow to uncraft ALL storage blocks, Either we need sth like "metal_blocks" or we need sth else.

Reply:
Not sure if a tag like that exists, and that would defeat the purpose of that change originally which was to allow someone to change diamond blocks back to diamonds (because I'm not sure you could classify diamonds as metal). Additionally, I think it would be helpful if someone could turn emerald blocks back to emeralds. So if you're opposed to all storage blocks, then perhaps I should just put in diamonds and emeralds directly into the blacksmith_products and leave it at that?

Reply:
Yeah, I'm fine with Diamond_block and emerald_block

Reply:
Waiting for those yet

Reply:
added diamond_block and emerald_block into the blacksmith_ingredient